src/plugins/intel_npu/src/compiler_adapter/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${SOURCES})
 add_library(${TARGET_NAME} STATIC ${SOURCES})
 
 ov_build_target_faster(${TARGET_NAME}
-    PCH PRIVATE "src/precomp.hpp"
```

Comment:
Specify them private or public it can be good option

Reply:
i don't see option where it can be useful right now. I suggest change this behavior in future if we face such case